pkg/kv/kvserver/store.go

```diff
@@ -1734,6 +1735,71 @@ func (s *Store) startGossip() {
 	}
 }
 
+var errSysCfgUnavailable = errors.New("system config not available in gossip")
+
+type shadowConfReader struct {
```

Comment:
this deserves commentary. 

It's not obvious that this thing belongs in this package. If it does, it definitely needs a more specific name that references spanconfig